conf/messages.cy

```diff
@@ -108,10 +108,12 @@ page.reg.accountingDates.help =	Nodwch ddyddiad o leiaf 3 diwrnod gwaith o heddi
 error.businessStartDate.required =	Nodwch pa bryd y bydd y cwmni’n dechrau masnachu
 page.reg.company-contact-details.description =	Rhowch un ffordd neu fwy o gysylltu â {0}
 page.reg.company-contact-details.label-two =	Cyfeiriad e-bost
+page.reg.company-contact-details.label-two-hint = Nodwch y cyfeiriad e-bost rydych chi am i CThEF ddefnyddio i gysylltu â chi.<br/> Gallai hyn fod yr un fath â chyfeiriad e-bost cofrestredig eich cwmni, ond nid oes rhaid iddo fod.
 page.reg.company-contact-details.label-three =	Rhif cyswllt
 page.reg.company-contact-details.label-three-hint =	Rhowch rif ffôn symudol, os oes un gennych
 page.reg.company-contact-details.label-four =	Rhif cyswllt arall
 page.reg.company-contact-details.validation.chooseOne =	Rhowch fanylion o leiaf un cyswllt ar gyfer Treth Gorfforaeth y cwmni
+page.reg.company-contact-details.helpMessage0 = Mae'r manylion cyswllt sydd wedi’u nodi ar y dudalen hon ar gyfer cyswllt CThEF yn unig.
```

Comment:
Page shows Maer manylion in Welsh and is missing an apostrophe.  It should show Mae'r manylion.  It looks like you've used a straight apostrophe, and not a curly one in the message file

Reply:
copy pasted from document, not sure how its missed. Hope this should be fine.